index.html

```diff
@@ -31,7 +31,139 @@ <h1>I’m Mariane
      <li><a href=""><img src="images/vector2.svg" alt=""></a></li>
     </ul>
    </section>
+   <section>
+    <article class="snapshoot"><img src="images/Snapshoot.png" alt=""></article>
+    <article>
+     <h1>Tonic</h1>
+     <div class="frame">
+      <h2>CANOPY</h2>
+      <div></div>
+      <p >Back End Dev</p>
+      <div></div>
+      <p>2015</p>
+     </div>
+     <div class="text">
+      <p>A daily selection 
+       of privately personalized reads; no accounts or sign-ups required.</p>
+     </div>
+     <div class="lang">
+      <p>html</p>
+      <p>css</p>
+      <p>javaScript</p>
+     </div>
+     <button type="button">See Project</button>
+    </article>
+   </section>
+   <section>
+    <article class="snapshoot"><img src="images/Snapshoot1.png" alt=""></article>
+    <article>
+     <h1>Multi-Post Stories</h1>
+     <div class="frame">
+      <h2>CANOPY</h2>
+      <div></div>
+      <p >Back End Dev</p>
+      <div></div>
+      <p>2015</p>
+     </div>
+     <div class="text">
+      <p>A daily selection 
+       of privately personalized reads; no accounts or sign-ups required.</p>
+     </div>
+     <div class="lang">
+      <p>html</p>
+      <p>css</p>
+      <p>javaScript</p>
+     </div>
+     <button type="button">See Project</button>
+    </article>
+   </section>
+   <section>
+    <article class="snapshoot"><img src="images/Snapshoot2.png" alt=""></article>
+    <article>
+     <h1>Tonic</h1>
+     <div class="frame">
+      <h2>CANOPY</h2>
+      <div></div>
+      <p >Back End Dev</p>
+      <div></div>
+      <p>2015</p>
+     </div>
+     <div class="text">
+      <p>A daily selection 
+       of privately personalized reads; no accounts or sign-ups required.</p>
+     </div>
+     <div class="lang">
+      <p>html</p>
+      <p>css</p>
+      <p>javaScript</p>
+     </div>
+     <button type="button">See Project</button>
+    </article>
+   </section>
+   <section>
+    <article class="snapshoot"><img src="images/Snapshoot3.png" alt=""></article>
+    <article>
+     <h1>Multi-Post Stories</h1>
+     <div class="frame">
+      <h2 class="tc">CANOPY</h2>
+      <div></div>
+      <p >Back End Dev</p>
+      <div></div>
+      <p>2015</p>
+     </div>
+     <div class="text">
+      <p>A daily selection 
+       of privately personalized reads; no accounts or sign-ups required.</p>
+     </div>
+     <div class="lang">
+      <p>html</p>
+      <p>css</p>
+      <p>javaScript</p>
+     </div>
+     <button type="button">See Project</button>
+    </article>
+   </section>
   </main>
-  <footer></footer>
+  <footer>
```

Comment:
- [ ] To keep your HTML semantic, you should consider removing the `about myself` section from the `<footer>`. Footers are usually the last section of the document, and typically contain authorship information, copyright information, contact information, sitemap, back to the top links, related documents. You can read more about footers [here](https://www.w3schools.com/tags/tag_footer.asp).